### ISSUE TITLE
Add XSMM implementations to benchmarks, par with reference

### ISSUE THREAD

**Issue by rengolin** [plaidml/tpp-mlir#197]
After #175 is fixed, we need to add the proper XSMM implementations to the benchmarks.

The `matmul` one is there (commented out), and we need to add at least `broadcast` and `relu` to have par with the reference implementation.

@hfp @alheinecke

Later on we should also add `convolution`, `softmax`, `maxpool`, some `normalization`, etc. (for another issue).

**Comment by rengolin** [comment]
We should probably submodule https://github.com/libxsmm/libxsmm-dnn and use that instead of home-brewing.

@alheinecke
